src/main/java/org/broadinstitute/hellbender/tools/funcotator/Funcotator.java

```diff
@@ -532,8 +549,9 @@ private void enqueueAndHandleVariant(final VariantContext variant, final Referen
                 .flatMap(List::stream)
                 .map(gf -> (GencodeFuncotation) gf).collect(Collectors.toList());
 
-
         //==============================================================================================================
+        // Create the rest of the funcotations:
+
```

Comment:
Extra blank line here.

Reply:
Actually, comments seem incorrect.  Please correct.

Reply:
I went over the comments and made them a little more descriptive.

I prefer to leave in the extra newline so that from a visual perspective the sections are a little more clear.  I made the dividers formatted the same way.